docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md

```diff
@@ -6,18 +6,29 @@ Integrate your B-L462E-CELL1 Discovery kit board along with the TYPE 1SE module
 
 - The B-L462E-CELL1/TYPE1SC board with a USB cable.
 - Installed **STM32CubeIDE**.
-- Installed **minicom** (for Linux) or RealTerm (for Windows) or other serial communication program.
+- Installed **minicom** (for Linux) or RealTerm or PuTTy (for Windows) or other serial communication program.
 - A user with access to the Coiote IoT Device Management platform.
 
-## Step 1: Cloning the Anjay freeRTOS client repository
+
+## Prepare binaries
+### Use an already built binary
+
+To get the latest binaries, navigate to [Anjay-freertos-client](https://github.com/AVSystem/Anjay-freertos-client/releases/).
+
+In this case we are going to use the Anjay-freertos-client-B-L462E-CELL1-TYPE1SC.bin file which after being downloaded must be added to B-L462E-CELL1/TYPE1SC board folder.
```

Comment:
`which after being downloaded must be added to B-L462E-CELL1/TYPE1SC board folder.` what do you mean here?